README.md

```diff
@@ -0,0 +1,46 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+## hello-rails-back-end
+Backend and Frontend created separately
+## Frontend
+The frontend part of this project: [hello-react-front-end](https://github.com/Kwentsir/hello-react-front-end)
+
+## Built With
+
+- Ruby
+- Ruby on Rails
+- PostgreSQL
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1- Copy this link `https://github.com/Kwentsir/hello-rails-back-end.git` <br>
+2- Get the directory that you want to clone the repository. <br>
+3- Open the command prompt in this directory. <br>
+4- Write `git clone https://github.com/Kwentsir/hello-rails-back-end.git` <br>
+5- Go to the repository folder in your command prompt `cd hello-rails-back-end`. <br>
+6- You might need to install bundles with `bundle install` and npm dependencies with `npm i`
+6- Go to command line and run `./bin/dev` for the live server. <br>
```

Comment:
In the README file for the frontend, you should provide a link to the Rails API (backend app).

Reply:
Hello @alexander16108, thanks for your feedback. 
I think the link is already in there on line five of the readme.